crates/markdown_preview/src/markdown_renderer.rs

```diff
@@ -152,7 +153,7 @@ fn render_markdown_heading(parsed: &ParsedMarkdownHeading, cx: &mut RenderContex
         .text_color(color)
         .pt(rems(0.15))
         .pb_1()
-        .child(render_markdown_text(&parsed.contents, cx))
+        .child(div().children(render_markdown_text(&parsed.contents, cx)))
```

Comment:
Can we just change this to a `children()` call?

```suggestion
        .children(render_markdown_text(&parsed.contents, cx))
```